components/BackHeader/BackHeader.module.scss

```diff
@@ -65,3 +65,12 @@
   display: flex;
   flex-direction: row;
 }
+
+.title {
+  overflow: hidden;
+  padding-right: 1.25rem;
+  text-align: right;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 250px;
```

Comment:
Please change this for a max-width, so short names aren't using more space than the needed